search.py

```diff
@@ -0,0 +1,70 @@
+# Name of this file: search.py
+# Location: at the root of the project
+# How to execute: python3 -m search.py query
+
+import json
+import sys
+import dotenv
+import louis.models.openai as openai
+import louis.db as db
+
+# This is used to load the .env file
+dotenv.load_dotenv()
+
+# Execute the SQL search function 
+def search(cursor, query_embedding):
```

Comment:
move the functions doing the interface between the python code (this file) and the database in louis.db.finesse

Reply:
Am I supposed to move this whole file or just the few lines that connect to the db ? I'm not sure

Reply:
No, just the functions that are reusable